en/flight_controller/snapdragon_flight_hardware_example_setup.md

```diff
@@ -0,0 +1,57 @@
+# Snapdragon Hardware Setup Example
+
+This guide documents in detail one particular build based on the Snapdragon Flight platform. The PX4 development team has built, tested and documented this specific setup.
+
+![](../../assets/hardware/snapdragon/snapdragon_minimal_finished.jpg)
+
+
+## PX4 Dev Team Recommended Setup
+
+We suggest using the Snapdragon Flight with the Lumenier QAV-R 250 frame and our custom hardware mounts. This setup uses a conventional PWM ESC board and therefore needs to be built with the `eagle_default` make target (see our instructions [here](https://dev.px4.io/en/setup/building_px4.html#qurt--snapdragon-based-boards)). 
+
+### Components
+* Snapdragon Flight: [available here](https://www.intrinsyc.com/vertical-development-platforms/qualcomm-snapdragon-flight/)
+* Stereo-vision add-on kit [available here](https://www.intrinsyc.com/vertical-development-platforms/qualcomm-snapdragon-flight/)
+* Frame: [Lumenier QAV-R 250](https://www.getfpv.com/qav-r-fpv-racing-quadcopter-5.html)
+* ESC: [Hobbywing 4in1 40A](https://www.getfpv.com/hobbywing-xrotor-micro-4-in-1-blheli-s-dshot600-esc.html)
+* Motors: [Lumenier RX2206 2350KV](https://www.getfpv.com/lumenier-rx2206-11-2350kv-motor.html)
+* Receiver: [Spektrum RC FPV Racing Serial Receiver](https://www.spektrumrc.com/Products/Default.aspx?ProdID=SPM4648)
+* Wi-Fi Antenna: [Laird Multiband](https://www.lairdtech.com/products/maf95056-nanoblade-internal-embedded-antenna-2400-2500-4900-6000-mhz)
+* GPS module (optional): [mRo GPS + Compass module](https://store.mrobotics.io/mRo-GPS-u-Blox-Neo-M8N-HMC5983-Compass-p/gps002-mr.htm)
+* Distance sensor (optional):[Trone range finder](http://www.teraranger.com/product/teraranger-one-distance-sensor-for-drones-and-robotics)
+* Distance sensor adapter (required only when using Trone): [I2C Adapter](http://www.teraranger.com/product/adapters-for-oneduo/)
+* 3D Parts: [Legs, Mounting brackets](https://drive.google.com/drive/u/0/folders/1MOunJae4ZLu6-bZ_-JvcPQUc9Aa5fe1U)
```

Comment:
What does 3D stand for here? I mean I guess they are 3 dimensional, but why are these two parts grouped like this with that name?

Reply:
Renamed to "Custom printed parts"